src/js/chapterEnd.js

```diff
@@ -0,0 +1,33 @@
+//import renderContent function. 
+
+//quiz will be a node that can be embedded into chapter end page. 
+const createChapterEnd = (chapterNumber, quiz, nextPage, homePage) => {
+    const base = document.createElement("div");
+    base.setAttribute('id', 'chapter-end')
+
+    let text = document.createElement("p");
```

Comment:
Also here as @Tudwall mentioned in the other end page: use const instead of let for html nodes.